ocaml/xapi/xapi_pool.ml

```diff
@@ -1397,12 +1397,12 @@ let certificate_install ~__context ~name ~cert =
 
 let install_ca_certificate = certificate_install
 
-let certificate_uninstall ~__context ~name =
+let uninstall_ca_certificate ~__context ~name ~force =
   let open Certificates in
-  pool_uninstall CA_Certificate ~__context ~name ;
+  pool_uninstall CA_Certificate ~__context ~name ~force ;
   Db_util.remove_ca_cert_by_name ~__context name
 
-let uninstall_ca_certificate = certificate_uninstall
+let certificate_uninstall = uninstall_ca_certificate ~force:false
```

Comment:
I've swapped the order of these definitions around to match message_forwarding.ml, which also allows us to pass ~force:false so that certificate_uninstall's is unchanged, does this make sense @psafont or was there a reason for them being this way around in xapi_pool.ml?

Reply:
the order of the bindings doesn't matter, whichever is more convenient for you, is good